helm-chart/binderhub/values.yaml

```
@@ -62,14 +62,35 @@ jupyterhub:
       BinderSpawner:
         auth_enabled: false
     extraConfig:
-      00-binder: |
-        from tornado import web
+      00-binderspawnermixin: |
+        """
+        Helpers for creating BinderSpawners
 
-        # image & token are set via spawn options
-        from kubespawner import KubeSpawner
+        This file is defined in helm-chart/binderhub/values.yaml and is copied to
+        binderhub/binderspawner_mixin.py by setup.py
```

With a migration plan, this is acceptable to me. Is this a reasonable understanding of the situation and a migration plan @yuvipanda @manics?

```suggestion
        FIXME: This file is defined in helm-chart/binderhub/values.yaml and is
               copied to binderhub/binderspawner_mixin.py by setup.py. To
               resolve this complexity, we have the following constraints to
               consider.
               
               1. This mixin is required both by the `binder` pod part of the
                  BinderHub Helm chart and the `hub` pod part of the JupyterHub
                  Helm chart.
               2. BinderHub's CI system can run locally against a JupyterHub
                  Helm chart installed in a k8s cluster, and then the code needs
                  to be available in the binderhub Python package and in the
                  JupyterHub Helm chart.
               
               A solution is to:

               1. Let the binderhub Python package contain this mixin
               2. Install the binderhub Python package in the hub pod's image
               3. Install the binderhub Python package in the binder pod's image
```